pkg/destroy/gcp/gcp.go

```diff
@@ -421,6 +421,10 @@ func (o *ClusterUninstaller) handleOperation(ctx context.Context, op *compute.Op
 		return fmt.Errorf("failed to delete %s %s: %w", resourceType, item.name, err)
 	}
 
+	if op == nil {
+		return fmt.Errorf("failed to handle operation that does not exist for resource %s", item.name)
+	}
```

Comment:
If the `op` is nil should we fail here or return `nil` (nothing to do)? Alternatively not call `handleOperation` if `op` is nil, but that would require changing a lot of calling sites, and I think handling here is probably simpler.